staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -76,3 +82,82 @@ func GlogSetter(val string) (string, error) {
 	}
 	return fmt.Sprintf("successfully set klog.logging.verbosity to %s", val), nil
 }
+
+const logLoggingFormat = "logging-format"
+
+var logRegistry *LogFormatRegistry = NewLogFormatRegistry()
+
+type LogFormatRegistry struct {
+	registry map[string]*logr.Logger
```

Comment:
Logger is already an interface, no need to have pointer to interface. Just use `map[string]logr.Logger`